packages/ios_platform_images/ios/ios_platform_images.podspec

```diff
@@ -15,11 +15,13 @@ Downloaded by pub (not CocoaPods).
   s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :http => 'https://github.com/flutter/packages/tree/main/packages/ios_platform_images' }
   s.documentation_url = 'https://pub.dev/packages/ios_platform_images'
-  s.source_files = 'Classes/**/*'
+  s.source_files = 'Classes/**/*.swift'
   s.dependency 'Flutter'
   s.platform = :ios, '11.0'
 
-  # Flutter.framework does not contain a i386 slice. Only x86_64 simulators are supported.
-  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
```

Comment:
This shouldn't be removed; I'll put a commit to re-add it (and remove the bridging header import in the unit test that was only necessary because of its removal).